config/hof-defaults.js

```diff
@@ -51,7 +51,8 @@ const defaults = {
     pdfConverter: process.env.PDF_CONVERTER_URL
   },
   serveStatic: process.env.SERVE_STATIC_FILES !== 'false',
-  sessionTimeOutWarning: process.env.SESSION_TIMEOUT_WARNING || 300
+  sessionTimeOutWarning: process.env.SESSION_TIMEOUT_WARNING || 300,
+  servicePaused: process.env.SERVICE_PAUSED || false
```

Comment:
reword to serviceUnavailable

Reply:
I think the boolean hof variables could have potential problems if we don't explicitly set it as proper boolean e.g. if you set it as 'false' that's actually truthy and could make the service go down unintentionally.  I think I know how to fix the issue in HOF.  I'll add it in my commits

Reply:
Agreed - I had to add another condition when before redirecting to avoid this, happy for you to add a fix in your commits